app/preprints/-components/submit/supplements/template.hbs

```diff
@@ -29,4 +29,23 @@
             {{t 'preprints.submit.step-four.create-button'}}
         </Button>
     </div>
+
+    {{#if this.displayExistingNodeWidget }}
+        <div>
+            <Preprints::-Components::Submit::Supplements::ExistingNodeWidget
+            @selectNode={{this.selectNode}}/>
+        </div>
+    {{/if}}
+
+    {{#if this.displayCancelButton}}
+        <div local-class='cancel-button-container'>
+            <Button
+                data-test-cancel-button
+                {{on 'click' this.onCancelProjectAction}}
+                class='btn btn-secondary btn-large'
```

Comment:
I don't think we should be using these bootstrap global classes anymore. `@type='secondary'` might be what you're looking for, but I think that's also the default type for `Button`s